lib/ansible/modules/cloud/amazon/glue_job.py

```diff
@@ -0,0 +1,377 @@
+    returned: when state is present
+    type: int
+    sample: 10
+command:
+    description: The JobCommand that executes this job.
+    returned: when state is present
+    type: complex
+    contains:
+        name:
+            description: The name of the job command.
+            returned: when state is present
+            type: string
+            sample: glueetl
+        script_location:
+            description: Specifies the S3 path to a script that executes a job.
+            returned: when state is present
+            type: string
+            sample: mybucket/myscript.py
+connections:
+    description: The connections used for this job.
+    returned: when state is present
+    type: dict
+    sample: "{ Connections: [ 'list', 'of', 'connections' ] }"
+created_on:
+    description: The time and date that this job definition was created.
+    returned: when state is present
+    type: string
+    sample: "2018-04-21T05:19:58.326000+00:00"
+default_arguments:
+    description: The default arguments for this job, specified as name-value pairs.
+    returned: when state is present
+    type: dict
+    sample: "{ 'mykey1': 'myvalue1' }"
+description:
+    description: Description of the job being defined.
+    returned: when state is present
+    type: string
+    sample: My first Glue job
+job_name:
+    description: The name of the AWS Glue job.
+    returned: always
+    type: string
+    sample: my-glue-job
+execution_property:
+    description: An ExecutionProperty specifying the maximum number of concurrent runs allowed for this job.
+    returned: always
+    type: complex
+    contains:
+        max_concurrent_runs:
+            description: The maximum number of concurrent runs allowed for the job. The default is 1. An error is
+                         returned when this threshold is reached. The maximum value you can specify is controlled by
+                         a service limit.
+            returned: when state is present
+            type: int
+            sample: 1
+last_modified_on:
+    description: The last point in time when this job definition was modified.
+    returned: when state is present
+    type: string
+    sample: "2018-04-21T05:19:58.326000+00:00"
+max_retries:
+    description: The maximum number of times to retry this job after a JobRun fails.
+    returned: when state is present
+    type: int
+    sample: 5
+name:
+    description: The name assigned to this job definition.
+    returned: when state is present
+    type: string
+    sample: my-glue-job
+role:
+    description: The name or ARN of the IAM role associated with this job.
+    returned: when state is present
+    type: string
+    sample: my-iam-role
+timeout:
+    description: The job timeout in minutes.
+    returned: when state is present
+    type: int
+    sample: 300
+'''
+
+from ansible.module_utils.aws.core import AnsibleAWSModule
+from ansible.module_utils.ec2 import camel_dict_to_snake_dict, ec2_argument_spec
```

Comment:
Since ec2_argument_spec is always added to AnsibleAWSModule you could remove this import and specify the module args in a dict (see lower section of https://github.com/ansible/ansible/blob/devel/lib/ansible/modules/cloud/amazon/GUIDELINES.md#porting-code-to-ansibleawsmodule).